react-frontend/src/pages/MakeOfferPage.js

```diff
@@ -1,37 +1,101 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { Container, Form, OfferGroup, Offer, HeaderLabel, MakeOfferButton } from '../authentication/AuthenticationStyling';
 import { useNavigate, useParams } from 'react-router-dom';
+import Talk from 'talkjs';
 import axios from 'axios';
 
+const TALKJS_APP_ID = 'tEOH06eA';
```

Comment:
Change this to an ENV Variable for security reasons

Reply:
I plan to. I had it there for testing to make sure I could get chat feature working. It's still not working yet so i have to still mess around with it a bit.